docs/stellar-core_example_validators.cfg

```diff
@@ -4,6 +4,7 @@ PUBLIC_HTTP_PORT=false
 NETWORK_PASSPHRASE="Example configuration"
 
 DATABASE="sqlite3://example.db"
+EXPERIMENTAL_PARALLEL_LEDGER_APPLY=false
```

Comment:
Out of curiosity, how does this change fit in here? This option is disabled by default, and it feels a little out of place to put an experimental flag in our example validator config.

Reply:
that's a good point. I added this in preparation to flip EXPERIMENTAL_PARALLEL_LEDGER_APPLY, since it's only supported on Postges. But I think it doesn't really belong in this PR, so I removed it for now.